src/service/project.ts

```diff
@@ -56,3 +57,34 @@ export const getLikedProjectsOfUser = async (userId: number) => {
     return item.proejct;
   });
 };
+
+export const getProject = async (projectId: number) => {
+  return await prisma.project.findFirst({
+    where: {
+      id: projectId,
+    },
+    include: {
+      owner: true,
+      ProjectView: true,
+      ProjectField: true,
+      ProjectImage: true,
+      ProjectMember: {
+        include: {
+          member: true,
+        },
+      },
+      ProjectSkill: true,
+      ProejctLike: true,
+    },
+  });
+};
+
+export const isLikedByUser = async (projectId: number, userId: number) => {
+  const result = await prisma.proejctLike.findFirst({
```

Comment:
#19 에 의거하여
```suggestion
  const result = await prisma.projectLike.findFirst({
```
로 수정하면 좋을 것 같습니다. : )

Reply:
관련된 모든 오타 수정 했습니다. :)